src/coprocessor/dag/rpn_expr/impl_arithmetic.rs

```diff
@@ -953,10 +959,11 @@ mod tests {
             (None, None, None),
         ];
 
+        let mut ctx = EvalContext::default();
         for (lhs, rhs, expected) in test_cases {
             let output = RpnFnScalarEvaluator::new()
-                .push_param(lhs.map(|f| Decimal::from_f64(f).unwrap()))
-                .push_param(rhs.map(|f| Decimal::from_f64(f).unwrap()))
+                .push_param(lhs.map(|f| f64_to_decimal(&mut ctx, f).unwrap()))
```

Comment:
Maybe we can rewrite these tests using string instead.. It would be better for a unit test to test less things other than itself.